new/LIT6299MGEgg.xml

```diff
@@ -0,0 +1,107 @@
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-12-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusText"/>
+                        <citedRange unit="page">43 (38th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusTrans"/>
+                        <citedRange unit="page">24–25 (38th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">37</citedRange>
+                        <citedRange unit="item">Ms 12, II.39 (38th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">726</citedRange>
+                        <citedRange unit="item">Vat 233, 3.38 (38th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6299MGEgg" passive="LIT4006Taamme"
+                    />
+                </listRelation>
+
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethb4"/>.</note>
+                    <ab>፴፰ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/> ወኮነ፡ በ፩ እመዋዕል፡ እንዘ፡ ይሥዕሉ፡ ሥዕላተ፡
+                        ወያሠረግው፡ ቤተ፡ ክርስቲያኑ፡ ለቅዱስ፡ ማር፡ ጊዮርጊስ፡ ወ፩፡ ብእሲ፡ ወሀበ፡ ዶርሆ፡</ab>
+                    <div type="textpart" subtype="explicit" xml:lang="gez">
+                        <note>This explicit is taken from <ref type="mss" corresp="BDLaethb4"
+                            />.</note>
+                        <ab>ወሠፈሩ፡ ኒኆ፡ ለውእቱ፡ ተስላስ፡ እምኀበ፡ ወድቀ፡ ወኮነ፡ ኑኁ፡ እምድር፡ ፭ እመተ፡ ሥዝረ፡ ወስፍሑ፡ ፫
```

Comment:
ኑኆ?

Reply:
Here simply nominative: ኑኁ፡

Reply:
sorry, I meant the first occurrence in this line (not very clear of me), ኑ instead of ኒ?